CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Update also draft order line total price after getting the unit price from plugin - #7080 by @IKarbowiak
 - Fix failing product tasks when instances are removed - #7092 by @IKarbowiak
 - Update GraphQL endpoint to only match exactly `/graphql/` without trailing characters - #7117 by @IKarbowiak
+- Add description to shipping method - #7116 by @IKarbowiak
```

Comment:
It looks like, we are changing an API a little bit here. Don't we want to add more details to changelog? Also, shouldn't be placed in breaking changes (I see that some types of input have been renamed/extended?